bookserver/routers/discuss.py

```diff
@@ -1,14 +1,63 @@
+# ******************************************************
```

Comment:
I love these docs!!! Very nice.

IMHO, don't manually wrap paragraphs -- that makes editing hard later. I think https://queue.acm.org/detail.cfm?id=1053354 makes a good case.

Reply:
but super hard to read in the editor.   I don't like having word wrap on for code.

Reply:
I like black in this case -- it word wraps code for me, then my editor word wraps the text. To each his own :).